### PR TITLE
bpo-41332: Added missing connect_accepted_socket() to AbstractEventLoop

### PR DESCRIPTION
**Pull request by agronholm**
https://bugs.python.org/issue41332

### CODE REVIEW

File: Lib/asyncio/events.py
Comment (aeros) [inline Lib/asyncio/events.py]: I'm aware that the typo was present in the original docstring, but this seems like a good opportunity to fix it while it's being moved.

```suggestion
        asyncio, but use asyncio to handle connections.
```

**Review by aeros — COMMENTED** [review]
Thanks for the PR @agronholm.

+1, `loop.connect_accepted_socket()` seems to me like it should be included in `AbstractEventLoop`, as it's important to have for asyncio interoperability purposes when used in addition to other libraries for handling connections. I think you are correct @agronholm that it was simply missed, but I'll wait on approval from @1st1 and/or @asvetlov to confirm this prior to merging.

Also, this should include an update to the What's New document for 3.10, in the "Porting to Python 3.10 > Changes in the Python API" sub-section since it's adding a method to an ABC. See the 3.9 What's New for an example: https://docs.python.org/3.9/whatsnew/3.9.html#changes-in-the-python-api (since the section doesn't exist yet in the 3.10 version).

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
Let's merge as-is.

**Pull request merged**
